CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Blip 
 
-# [1.12.5-dblp.0.2.0] - 2019-03-19 
+# [1.12.5-dblp.0.2.2] - 2019-03-19 
```

Comment:
is it the proposed new version? 1.12.5-dblp.0.2.2 ?

Reply:
yes